internal/webhook/external/providers/github/provider.go

```diff
@@ -1,19 +1,12 @@
 package github
 
 import (
-	"crypto/hmac"
-	"crypto/sha256"
-	"encoding/hex"
-	"encoding/json"
 	"errors"
 	"fmt"
 	"net/http"
 	"os"
-	"strings"
 
-	"github.com/akuity/kargo/internal/git"
-	xhttp "github.com/akuity/kargo/internal/http"
-	"github.com/akuity/kargo/internal/webhook/external/events"
+	gh "github.com/google/go-github/github"
```

Comment:
This is old. You want this:

```suggestion
	gh "github.com/google/go-github/v71/github"
```

Reply:
Will need to re-run `go mod tidy` after.

Reply:
Nevermind... I was somehow looking at out of date diffs when I wrote this.